src/Redux/userSlice.js

```diff
@@ -15,7 +15,7 @@ export const fetchUser = createAsyncThunk('user/fetchUser', async () => {
     });
```

Comment:
getting rid of an error message (if no user is detected "no user" will be stored in the redux store)